src/tool_help.c

```diff
@@ -135,13 +137,13 @@ static void get_categories(void)
 void tool_help(char *category)
 {
   unsigned int cols = get_terminal_columns();
-  puts("Usage: curl [options...] <url>");
   /* If no category was provided */
   if(!category) {
     const char *category_note = "\nThis is not the full help, this "
       "menu is stripped into categories.\nUse \"--help category\" to get "
       "an overview of all categories.\nFor all options use the manual"
       " or \"--help all\".";
```

Comment:
Maybe we should mention you can also get help for specific cmdline options? Will help discoverability

Reply:
:+1: yes, good idea